docker/develop/README.md

```diff
@@ -2,10 +2,9 @@
 
 Navigate to this folder and run:
 ```
-docker build -t hyperledger/iroha-docker-develop:v1 .
+docker build -t iroha-docker-develop .
```

Comment:
https://github.com/hyperledger/iroha/blob/2d993dc9aa8f4caabb88694ad8283d19ffdc806d/docker/docker-compose.yml#L5
It may be more convenient for user to use the suggested tag and call `scripts/run-iroha-dev.sh`
```suggestion
docker build -t hyperledger/iroha:develop-build .
```

Reply:
Reverted to initial state